package.json

```diff
@@ -27,11 +27,12 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
+    "lodash.get": "^4.4.2",
```

Comment:
This dependency add 4.5Kb to the bundle (https://bundlephobia.com/result?p=lodash.get@4.4.2).
Maybe you could just add a function like this one https://gist.github.com/Aschen/6ba04c9eca3051a83f7e1dbddf38c6a1

Reply:
:+1: